src/controllers/__tests__/messagesController.test.ts

```diff
@@ -324,6 +324,7 @@ describe("MessagesController#getMessage", () => {
 
     req.user = mockedUser;
     req.params = { id: anId };
+    req.query = { public_message: "true" };
```

Comment:
Since it's an optional parameter, I'd rather add a new test having public_message parameter defined.